data/poweredBy/orgs/OntaOnta.yaml

```diff
@@ -0,0 +1,14 @@
+category: comorg
+demoUrl: ''
+description: 
+<p>OntaOnta is a fully-integrated online teaching and learning system created
```

Comment:
I think this line needs to be indented two spaces to fix the [automated build failed](https://github.com/Pylons/trypyramid.com/pull/305/checks?check_run_id=2479223907). There might be other syntax issues, but let's start with this one.

Also you do not need to wrap it with `<p>` tags.

Reply:
Build succeeds now. Thank you!